static/templates/keyboard_shortcuts.hbs

```diff
@@ -220,6 +220,10 @@
                     <td class="definition">{{t 'Edit selected message or view message source' }}</td>
                     <td><span class="hotkey"><kbd>E</kbd></span></td>
                 </tr>
+                <tr id="move-message-hotkey-help">
+                    <td class="definition">{{t 'Move selected message and (optionally) other messages in this topic' }}</td>
```

Comment:
One option is something like "Move message or topic".

Reply:
Hmm, maybe "Move messages or topic"? The default behavior is to move some, but not all, messages in the topic.

Reply:
Yeah that works for me.